app/helpers/ex_link_helper.rb

```diff
@@ -0,0 +1,20 @@
+module ExLinkHelper
+  # instead of boring true/false for booleans renders appropriate symbols
+  def bool2icon(boolval)
+    if boolval
+      res = "<i class='fa fa-check'>"
```

Comment:
There is a helper called `fa_icon('check')` which handles html_safe and such for our favicon.